webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -74,11 +79,10 @@ const EmptyStatePattern = props => {
 EmptyStatePattern.propTypes = emptyStatePatternPropTypes;
 
 EmptyStatePattern.defaultProps = {
-  icon: 'add-circle-o',
+  icon: <PlusCircleIcon />,
```

Comment:
Since its a default prop, will it ever get to this condition? `const iconElement = icon || <PlusCircleIcon size="2x" />;`